ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -139,7 +139,9 @@ images:
       - registry.ci.openshift.org/openshift/release:golang-1.20
   to: assisted-service-el8
 - dockerfile_literal: |
+    FROM quay.io/multi-arch/yq:4.30.5 as yq4
     FROM coreos/centos
+    COPY --from=yq4 /usr/bin/yq /usr/bin/yq
```

Comment:
I'm not sure if this will affect other jobs. Since many jobs use this image, I prefer not to modify it. If needed, we can use jq or install yq in the steps.

Reply:
Hi @LiangquanLi930 I already got in touch with the owners of the image and they have already approved it. 
This is not a harmful change and in general installing yq/jq in a temp folder is a practice we should not encourage.

Reply:
Ref commit with explanation: https://github.com/openshift/release/pull/54483/commits/1dddda1c87765310aa90dae5aadab6c22518b501

Reply:
Ref commit with explanation: https://github.com/openshift/release/pull/54483/commits/b3fcae4d45dc473d2f455fa70d16276b448654b5